src/Tools/Md2Html/Program.cs

```diff
@@ -82,15 +89,19 @@ static string GetData()
         static bool CheckForHelp(string[] args)
         {
             bool help = false;
-            var options = new OptionSet()
-            {
-                {@"h|?|help", @"Show help and exit", v => help = v != null },
-            };
+            var parser = new Parser(options => {
+                options.IgnoreUnknownArguments = true; options.HelpWriter = Console.Out;
+                options.CaseSensitive = false;
+            });
+            var results = parser.ParseArguments<CMDLineOptions>(args);
```

Comment:
guess it's also possible that this is somehow messing with Console.Out or modifying the input argument list so it never makes its way to the rest of the Main function?